bio/quast/wrapper.py

```
@@ -67,9 +71,61 @@
+    ### Saving LOG files
+    save_output(f"{tmpdir}/quast.log", snakemake.log.get("quast"))
+    for ext in ["out", "err"]:
+        save_output(
+            f"{tmpdir}/contigs_reports/contigs_report_genome.std{ext}",
+            snakemake.log.get(f"contigs_{ext}"),
+        )
+
+    ### Saving OUTPUT files
+    # Report files
+    for ext in ["html", "pdf", "tex", "txt", "tsv"]:
+        save_output(f"{tmpdir}/report." + ext, snakemake.output.get(f"report_{ext}"))
```

Ruff

Undefined name `snakemake`

(F821)